user-access-event-handlers/src/main/java/no/unit/nva/useraccess/events/service/UserMigrationService.java

```diff
@@ -1,9 +1,8 @@
 package no.unit.nva.useraccess.events.service;
 
-import no.unit.nva.useraccessmanagement.dao.UserDb;
+import no.unit.nva.useraccessmanagement.model.UserDto;
 
 public interface UserMigrationService {
 
-    UserDb migrateUser(UserDb user);
-
+    UserDto migrateUser(UserDto user);
```

Comment:
migrationService should not be aware of the database objects (UserDb). It should instead operate on the public model (DTOs)